mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/chargedialog/ChargeDialogFragment.java

```diff
@@ -198,7 +197,8 @@ public void showChargesCreatedSuccessfully(Charges changes) {
 
     @Override
     public void showFetchingError(String s) {
-        Toaster.show(rootView, s);
+        Toast.makeText(getActivity(), s , Toast.LENGTH_LONG)
```

Comment:
This should not be a toast. It should be a snackbar attached to the background view to conform to the theme.